test/test_overrides.py

```diff
@@ -629,53 +629,65 @@ def instance_gen():
 
         func_args = []
         is_method = is_tensor_method_or_property(func)
+
+        def _simple_type_parser(func, arg):
+            # Guess valid input to aten function based on type of argument
+            t = arg['simple_type']
+            if t.endswith('?'):
+                t = t[:-1]
+            if t == 'Tensor':
+                return instance_gen()
+            elif t == 'TensorList' or t == 'ITensorListRef':
+                return [instance_gen(), instance_gen()]
+            elif t == 'c10::List<c10::optional<Tensor>>':
+                return [instance_gen(), instance_gen()]
+            elif t == 'IntArrayRef' or t == 'SymIntArrayRef':
+                size = arg.get('size', 2)
+                if size == 1:
+                    return 1
+                else:
+                    return [1] * size
+            elif t == 'Scalar':
+                return 3.5
+            elif t == 'bool':
+                return False
+            elif t == 'Dimname':
+                return ""
+            elif t == 'DimnameList':
+                return [""]
+            elif t.startswith('int'):
+                return 0
+            elif t in {'Stream'}:
+                return torch.Stream()
+            elif t.startswith('float') or t == 'double':
+                return 1.0
+            elif t in {'Generator', 'MemoryFormat', 'TensorOptions'}:
+                return None
+            elif t == 'ScalarType':
+                return torch.float32
+            elif t == 'c10::string_view':
+                return ""
+            elif t == 'SymInt':
+                # TODO: generate actual SymbolicInt
+                return 1
+            else:
+                raise RuntimeError(f"Unsupported argument type {t} for {arg['name']} of function {func}")
+
         if func in annotated_args:
             for arg in annotated_args[func]:
                 # Guess valid input to aten function based on type of argument
                 t = arg['simple_type']
                 if t.endswith('?'):
                     t = t[:-1]
```

Comment:
How come you need to do this part twice now?

Reply:
Addressed